### ISSUE TITLE
Leprechaun + Salamander synergy does not yield Steel Bars

### ISSUE THREAD

**Issue by gutblender** [MelvorIdle/melvoridle.github.io#1168]
I didn't notice until much later, and granted my method of having discovered this is a bit dubious.

While working on thieving masteries, I elected to use Leprechaun + Salamander to help with Smithing masteries afterward. I yielded about 10k of each bar this way. I know this because I had about 10k bronze bars in my bank having had none prior. I didn't at that time notice I had no steel bars. But now I see that in my completion log, I have encountered about 15k steel bars ever, and have about 14750 now that I just finished smelting, and I know I had none before. Furthermore I have sold basically no steel bars ever (35). 
To my knowledge I have only smithed steel items to cross them off the item log, and the steel weapons I have on the log are from pickpocketing the Knight to 99 mastery.

I have no idea whether other bars might be similarly affected. 
Sorry for the terrible rambling explanation. I can attempt to reproduce the issue shortly by time-skipping.

I will export and attach my save.
[melvoridle_export_LeprechaunSalamanderNoSteelBars.txt.txt](https://github.com/MelvorIdle/melvoridle.github.io/files/6964599/melvoridle_export_LeprechaunSalamanderNoSteelBars.txt.txt)

**Comment by gutblender** [comment]
I've found that the bar received appears to be related to the ore, similar to how the Mole + Salamander synergy works. I like this mechanic, however this is not how the synergy describes itself. Leprechaun + Salamander says a *random* bar. This is clearly not random; I just started thieving the Miner on my hardcore, which had zero iron/copper/tin nor iron/bronze bars beforehand. 
![image](https://user-images.githubusercontent.com/43554226/130554244-e09c42f4-4261-449d-bc1a-c281d96f0851.png)
Naturally I receive more copper/tin than iron, but so do I receive more bronze bars than iron, which is great but it's not as the synergy describes itself.
One idea to fix this is to merely update the synergy description to match Mole + Salamander. If you do this, I'd humbly suggest considering awarding a Steel Bar when Coal Ore is pickpocketed from the Miner.
